libraries/AP_Scripting/examples/plane-doublets.lua

```diff
@@ -30,7 +27,6 @@ local K_RUDDER = 21
 local start_time = -1
 local end_time = -1
 local now = -1
-local desired_mode = -1
```

Comment:
libraries/AP_Scripting/examples/plane-doublets.lua:32:7: variable desired_mode is never accessed